astropy/coordinates/builtin_frames/intermediate_rotation_transforms.py

```diff
@@ -116,8 +117,9 @@ def tete_to_gcrs(tete_coo, gcrs_frame):
     rbpn = erfa.pnm06a(jd1, jd2)
     newrepr = tete_coo.cartesian.transform(matrix_transpose(rbpn))
 
-    gcrs = GCRS(newrepr, obstime=tete_coo.obstime, obsgeoloc=tete_coo.obsgeoloc,
-                obsgeovel=tete_coo.obsgeovel)
+    obsgeoloc, obsgeovel = tete_coo.location.get_gcrs_posvel(tete_coo.obstime)
+    gcrs = GCRS(newrepr, obstime=tete_coo.obstime,
+                obsgeoloc=obsgeoloc, obsgeovel=obsgeovel)
 
     # now do any needed offsets (no-op if same obstime and 0 pos/vel)
```

Comment:
May as well fix this comment also. Drop the zero.

Reply:
Done